device-registry/src/main/scala/org/genivi/sota/device_registry/Boot.scala

```diff
@@ -88,8 +88,9 @@ object Boot extends BootApp with Directives with BootMigrations
     db.run(for {
       device <- DeviceRepository.findByUuid(msg.device)
       status = currentDeviceStatus(device.lastSeen, Seq((Instant.now(), UpdateStatus.withName(msg.status))))
-      _ <- DeviceRepository.setDeviceStatus(device.uuid, status)
-    } yield ())
+      res    <- DeviceRepository.setDeviceStatus(device.uuid, status)
+      _      = messageBus.publish(DeviceUpdateStatus(device.namespace, device.uuid, status))
+    } yield res)
```

Comment:
Move this to some `daemon/DeviceUpdateStatusChanger.scala` or whatever you want to call it:

The code should probably look something like this.
```
  private def updateDeviceStatus(msg: UpdateSpec) = {
    import org.genivi.sota.data.DeviceSearchCommon._
    db.run(for {
      device <- DeviceRepository.findByUuid(msg.device)
      status = currentDeviceStatus(device.lastSeen, Seq((Instant.now(), UpdateStatus.withName(msg.status))))
      res    <- DeviceRepository.setDeviceStatus(device.uuid, status)
    } yield (device, status)).flatMap{ case (device, status) =>
      messageBus.publish(DeviceUpdateStatus(device.namespace, device.uuid, status))
    }
  }
```